doc/events.md

```diff
@@ -3,55 +3,58 @@ Events
 To see all operations available on events:
 ```
 origo events -h
-origo event_streams -h
 ```
 
 TOC:
-* [What is an event](#what-is-an-event)
-* [Create event stream](#create-event-stream)
-* [Event status](#event-status)
-* [Delete event stream](#delete-event-stream)
-* [Sending events](#sending-events)
-* [Consuming events](#consuming-events)
+- [Events](#events)
+- [What is an event](#what-is-an-event)
+- [Create event streams](#create-event-streams)
+- [Event status](#event-status)
+- [Delete event stream](#delete-event-stream)
+- [Sending events](#sending-events)
+  - [Single json events](#single-json-events)
+  - [Sending multiple json events](#sending-multiple-json-events)
+- [Sinks](#sinks)
+- [Subscribable event streams](#subscribable-event-streams)
 
 # What is an event
 Documentation is available on [github](https://oslokommune.github.io/dataplattform/)
 
 
-# Create Event Streams
+# Create event streams
 An event stream is connected to a [dataset](datasets.md), and before you can continue with events you need to [create a dataset](datasets.md#create-dataset) and a [version](datasets.md#create-version) for that dataset. If this already exist you can continue with the following steps:
 
 Create:
 ```bash
-origo event_streams create <datasetid> <versionid>
+origo events create-stream <dataset-uri>
 ```
 
 # Event status
-A stream status must be ACTIVE before you can start sending events to it. To poll for the status manually:
+A stream status must be `ACTIVE` before you can start [sending events](#sending-events) to it. To poll for the streams status manually:
 ```bash
-origo event_streams ls <datasetid> <versionid>
+origo events cat <dataset-uri>
```

Comment:
Jeg er litt usikker på om `cat` er riktig navn på kommandoen... Kan forveksles med å hente ut eventer? Akkurat her er kanskje `status` bedre? Men kan ta en diskusjon på det og evt. endre seinere :-)

Reply:
Hm, ja - men tenker du spesifikt her fordi denne returnerer _flere_ ressurser (som `datasets ls <datasetid>`), eller generell bruk av `cat` (jf. "best practices")?

Reply:
Jeg tenker at Unix-kommandoen `cat` viser innholdet i en fil (datasett, eventstrøm, ...), ikke metadata om den... Så en revurdering av "best practice" ja. Tror analogien ble litt dårlig for akkurat denne kommandoen, men har ikke noe veldig godt alternativt forslag.

Hva konkret gjør denne kommandoen? Viser den all konfigurasjon og status for eventstrømmen?

Reply:
Ja, rimer ikke helt. Nåværende `cat` returnerer metadata om eventstrøm og tilhørende "sub-ressurser" (på tilsvarende måte som `datasets ls <datasetid>` => `{"dataset": ..., "versions": ..., "latest": ...}`).

Kan nok være `show` er bedre for enkelte ressurser. Ser aws cli bruker `describe-*` - riktignok for "_en eller flere ressurser_", f.eks. [`describe-vpcs [--vpc-ids <v>][--filters <v>]`](https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-vpcs.html). I så fall kanskje begrense til entallsform `describe-stream <dataset-uri>` (om det ikke blir aktuelt å kunne hente ut metadata for flere samtidig, utover `ls`)? 

```
{
    "stream": {"status": "ACTIVE",  ...},
    "subscribable": {"status": "INACTIVE", ...},
    "sinks": [
        {"status": "ACTIVE",  ...}
    ]
}
```
```
Event stream: dataset/1
+-----------+------+-----------------+---------+--------+------------+----------------------------------+
| Id        | Raw  | Confidentiality | Deleted | Status | Updated by | Updated date                     |
+-----------+------+-----------------+---------+--------+------------+----------------------------------+
| dataset/1 | True | green           | False   | ACTIVE | janedoe    | 2020-08-26 08:07:21.903672+00:00 |
+-----------+------+-----------------+---------+--------+------------+----------------------------------+


Subscribable for event stream:
+---------+----------+---------------+----------------------------------+
| Enabled | Status   | Updated by    | Updated date                     |
+---------+----------+---------------+----------------------------------+
| False   | INACTIVE | janedoe       | 2020-08-20 06:51:42.786699+00:00 |
+---------+----------+---------------+----------------------------------+


Sinks for event stream:
+-------+---------------+--------+------------+----------------------------------+
| Id    | Type          | Status | Updated by | Updated date                     |
+-------+---------------+--------+------------+----------------------------------+
| a1b2c | elasticsearch | ACTIVE | janedoe    | 2020-08-26 08:07:21.378438+00:00 |
+-------+---------------+--------+------------+----------------------------------+
```

Reply:
Hva med bare `origo events describe <dataset-uri>`?

Reply:
AWS har f.eks. `aws kinesis describe-stream --stream-name <name>` - men synes ofte de AWS-kommandoene blir veldig verbose...

Tror egentlig jeg er mer tilhenger av `show` - kortere og enklere enn `describe`. Selv om kanskje sistnevnte er mer beskrivende (d'oh!)...